api/server/server.go

```diff
@@ -537,15 +537,19 @@ func (s *Server) Close() {
 	}
 }
 
-func createDatastore(conf Config) (datastore.TxnDatastore, error) {
+func createDatastore(conf Config, longTimeout bool) (datastore.TxnDatastore, error) {
```

Comment:
We enable a new flag `longTimeout` which returns a datastore that has long timeouts for operations and transactions. 
Basically, the datastore that Powergate uses for operating has _false_ value. For migrations, we create use a _true_ value to allow for bigger timeouts since we do long-running operations.